### PR TITLE
docker/android-base: simpler Dockerfile

### PR DESCRIPTION
**Pull request by proppy**
- use java:openjdk-6-jdk
- use curl
- remove Vagrant user
- add gradle and ant
- install in `/usr/local`
- set `ANDROID_HOME`

### CODE REVIEW

File: docker/android-base/Dockerfile [hunk 1]
```diff
@@ -1,59 +1,20 @@
-FROM ubuntu:14.04
+FROM java:openjdk-6-jdk
```
Comment (samtstern) [inline docker/android-base/Dockerfile]: Have you tried running the android-studio after making this change to android-base? I remember Android Studio yelled at me for trying to use openjdk6.

**Comment by samtstern** [comment]
All looks good assuming you test android-studio image on top and confirm it still works.

**Comment by proppy** [comment]
rebased

**Comment by samtstern** [comment]
I'm good with all of these changes, do you want me to merge or are you still working on it?

**Pull request merged**